### ISSUE TITLE
Fix request: false positive in HttpClientBuilderWithoutTimeouts

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#160]
example:
```
class HttpClientGood {
    private ClientHttpRequestFactory getClientHttpRequestFactory() {
        return new HttpComponentsClientHttpRequestFactory(createHttpClient()); // false positive
    }

    private CloseableHttpClient createHttpClient () {
        RequestConfig requestConfig = RequestConfig.custom()
                .setConnectionRequestTimeout(350)
                .setConnectTimeout(250)
                .setSocketTimeout(4000)
                .build(); // good, all timeouts set

        return HttpClientBuilder.create()
                .setDefaultRequestConfig(requestConfig) // good, all timeouts set, if missing > default timeouts?
                .build();
    }
}
```

**Comment by BertKoor** [comment]
duplicate issue?
https://github.com/jborgers/PMD-jPinpoint-rules/issues/163
